### PR TITLE
Add new license to install script

### PR DESCRIPTION
**Pull request by raskchanky**
For background, see https://blog.chef.io/2019/04/02/chef-software-announces-the-enterprise-automation-stack

PLEASE NOTE: I've added the DO-NOT-MERGE label to this for now because this PR should not merge until 4/29.

Signed-off-by: Josh Black <raskchanky@gmail.com>

### CODE REVIEW

File: LICENSE-NOTICE [hunk 1]
```diff
@@ -0,0 +1,5 @@
+CHEF BINARY DISTRIBUTION AGREEMENT
+
+Use of this distribution of Habitat is subject to the terms and conditions of the Chef Binary Distribution Agreement. You can find the latest copy of the agreement here:
```
Comment (btm) [inline LICENSE-NOTICE]: Can we change this line and the top one to "Chef End User License Agreement" ?

Comment (raskchanky) [reply LICENSE-NOTICE]: Yes sir

File: install.sh [hunk 2]
```diff
@@ -10,9 +10,12 @@ sudo () {
 type curl >/dev/null 2>&1 || { echo >&2 "curl is required for installation of habitat, but was not found. Exiting."; exit 1; }
 
 curl https://raw.githubusercontent.com/habitat-sh/on-prem-builder/master/NOTICE
+echo
 
+curl https://raw.githubusercontent.com/habitat-sh/on-prem-builder/master/LICENSE-NOTICE
 echo
-read -r -p "Continue with installation? [y/N] " response
+
+read -r -p "Do you accept the terms of this license? Answering yes will proceed with the installation. [y/N] " response
```
Comment (predominant) [inline install.sh]: Is there some way to automate this response?

Comment (raskchanky) [reply install.sh]: Not currently, but that's a great idea.  I'll add that before this merges.

**Pull request merged**